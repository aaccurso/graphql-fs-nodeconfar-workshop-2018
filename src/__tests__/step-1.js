const { graphql } = require('graphql');
const schema = require('../schema');

describe('step 1', () => {
  it('should query files', async () => {
    const source = `
      query files {
        files {
          name
          type
        }
      }
    `;

    expect(await graphql({ schema, source })).toMatchSnapshot();
  });
});
