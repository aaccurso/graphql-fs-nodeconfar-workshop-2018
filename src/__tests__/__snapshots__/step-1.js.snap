// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`step 1 should query files 1`] = `
Object {
  "data": Object {
    "files": Array [
      Object {
        "name": "Bart_Simpson.png",
        "type": "File",
      },
      Object {
        "name": "Lisa_Simpson.png",
        "type": "File",
      },
      Object {
        "name": "Maggie_Simpson.png",
        "type": "File",
      },
    ],
  },
}
`;
